go/staking/api/api.go

```diff
@@ -29,6 +29,10 @@ const (
 )
 
 var (
+	// RewardFactorEpochSigned is the factor for a reward
+	// distributed per epoch to entities that have signed 3/4 of the blocks.
+	RewardFactorEpochSigned *quantity.Quantity
```

Comment:
When would you change this to something different from 1? Couldn't the same be achieved by lowering the reward amount?

Reply:
to control the relative sizes of rewards. currently this one is 1 and the one for election eligibility is 0. this "what you did" factor is multiplied by the reward schedule factor. and then later by the amount staked

Reply:
Oh I see we didn't remove the scheduler-based rewards. Any reason why we should keep both?

Reply:
- so we can support the scheduler one later, if we want to.

.......... or anyone using this codebase. if anyone's looking, this is not a statement about governance :speak_no_evil:

Reply:
oops, gotta update that code comment too now that it's not hardcoded to 3/4